### PR TITLE
merge 1.2 release from develop to master

### PR DESCRIPTION
**Pull request by saupchurch**

### CODE REVIEW

**Review by hitz — APPROVED** [review]

**Review by jb-adams — COMMENTED** [review]
Looks great, happy to see that threshold filtering has returned.

One comment would be that is it clear what units `featureMinParam` and/or `featureMaxParam` are supplied in if `unitsParam` isn't provided? In other words, if `unitsParam` isn't supplied, what units are feature min/max? Not sure if I skimmed too quickly and missed this.

I'll hold off on approving until #101 has been approved so I can see everything that will be closed to `master`.

**Pull request merged**
